tests/integration/run_tests.py

```diff
@@ -663,10 +664,13 @@ async def test_crud_folders_works(sess):
     # Publish the folder
     folder_id = await publish_folder(sess, folder_id)
     async with sess.get(f"{folders_url}/{folder_id}") as resp:
+        LOG.debug(resp.status)
```

Comment:
Integration test fails here when calling `publish_folder()`. However this bit works for me locally with both actual DataCite API, as well as the mocked DOI app. So most likely the docker compose file causing errors and not running the mock app properly atm. Also line 667 can be removed